metadata.go

```diff
@@ -0,0 +1,104 @@
+package tfjson
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/go-version"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// MetadataFunctionsFormatVersionConstraints defines the versions of the JSON
+// metadata functions format that are supported by this package.
+var MetadataFunctionsFormatVersionConstraints = ">= 0.1, < 2.0"
```

Comment:
```suggestion
var MetadataFunctionsFormatVersionConstraints = "~> 1.0"
```
AFAICT the first version will be `1.0` and we'd expect breaking changes in 2.0 per semver, right?